frontend/src/pages/Home/components/JoinUsForm/Form.jsx

```diff
@@ -1,11 +1,17 @@
 import React, { useState } from "react";
 import Joi from "joi-browser";
+<<<<<<< HEAD
```

Comment:
you haven't resolved the merge conflicts
resolve those and request for review

Reply:
I have tried to do that but Idk why they aren't being reflected
Leme try that again